### PR TITLE
Implement January Seasonal, Polar Opposites

### PR DESCRIPTION
**Pull request by ascriptmaster**
Mono-Ice, Inverse Battles.

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -1521,7 +1521,7 @@ exports.commands = {
 			/* falls through */
 		default: {
 			if (!Config.groups[target]) {
-				return this.errorReply("The rank '" + target + '" was unrecognized as a modchat level.');
+				this.errorReply("The rank '" + target + '" was unrecognized as a modchat level.');
```
Comment (Joimer) [inline commands.js]: Sneaky

**Comment by Joimer** [comment]
The seasonal seems good, @Zarel ?

File: commands.js (hunk 1 above)
Comment (ascriptmaster) [inline commands.js]: npm was failing, so...

**Comment by ascriptmaster** [comment]
Removed the fix in commands.js, since Zarel already made a commit to fix it.

**Comment by Zarel** [comment]
I blame the `npm test` issue on the fact that ESLint no longer supports iojs 2.x, leaving me no way of testing server changes before pushing them. :|

**Comment by Zarel** [comment]
I don't really pay attention to seasonals, so you guys will need to make this decision without me.

**Comment by TheImmortal** [comment]
Forgot to change the alias

**Comment by ascriptmaster** [comment]
Ah, yes. Aliases are important.

**Comment by ascriptmaster** [comment]
If there are no more comments I'm going to merge this in a few hours once I get to a computer again, or something.

Maybe tomorrow morning.

**Pull request merged**